.github/workflows/ci.yml

```diff
@@ -156,6 +168,12 @@ jobs:
         with:
           submodules: recursive
 
+      - name: Setup SSH agent
+        if: contains(env.RUSTFLAGS, 'sentry')
```

Comment:
Wonder if we should just check here for the existence of the secret. Might lead to better errors, when the pull fails and/or easier to understand the action?